astropy/wcs/utils.py

```diff
@@ -1196,7 +1196,7 @@ def fit_wcs_from_points(
         wcs.wcs.cd = wcs.wcs.pc
         wcs.wcs.__delattr__("pc")
 
-    if (type(sip_degree) != type(None)) and (type(sip_degree) != int):
+    if (sip_degree is not None) and (type(sip_degree) != int):
```

Comment:
Let's ping @mcara in case there is something subtle that would require comparing `type` directly like that.

Reply:
@mcara, does this look good?